### PR TITLE
use pub-sub instead of FlowMessaging

### PR DESCRIPTION
**Pull request by rahul-daga-imgtec**
Signed-off-by: Rahul Daga <Rahul.Daga@imgtec.com>

### CODE REVIEW

File: src/flow_interface.c [hunk 1]
```diff
@@ -244,15 +244,16 @@ static bool GetUserId(char *userId)
  */
 bool SendMessage(char *message)
```
Comment (abhijit-mahajani-imgtec) [inline src/flow_interface.c]: Change the function name to PublishStatus or PublishMessage

**Comment by abhijit-mahajani-imgtec** [comment]
Need to change the prints

Message sent to user = 12:32:13 29-03-2016 LED on

Its status published by device, not message sent to user

**Comment by abhijit-mahajani-imgtec** [comment]
Approved.

**Comment by nikhil-zinjurde-imgtec** [comment]
Verified under Device Topics -> DeviceStatus, but seeing many Publish messages getting lost

**Pull request merged**